src/styles/blocks/catalog.scss

```
@@ -17,9 +17,16 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: 300ms linear;
+    
 
     &:not(:nth-child(4n)) {
       margin-right: 48px;
     }
+
+    &:hover {
+      transform: scale(1.2);
+      color:#34568b;
```

don't forget to create variables for all colors
```suggestion
      color: $c-color;
```